README.md

```diff
@@ -53,30 +65,34 @@ The `make` command will get all of DKAN's dependencies _including_ Drupal core.
 Finally, let's install DKAN.
 
 ```bash
-dktl dkan:install
+dktl install
 ```
 
 You can find your local site URL by typing `dktl docker:surl`.
 
-## Existing DKAN Site to DKTL
+## Structure of a DKAN-Tools-based project
 
-One of the many reasons for using DKTL is to create a clear separation between **our** application, and other software that we are simply using. To accomplish this, we want as much of what makes our application unique to live in the ``src`` directory.
+One of the many reasons for using DKTL is to create a clear separation between the code specific to a particular DKAN site (i.e. "custom code") and the dependencies we pull in from other sources (primarily, DKAN core and Drupal core). 
 
-To get started we find what version of DKAN our current application is using, and run the following commands:
+To accomplish this, DKAN Tools projects will have the following basic directory structure, created when you run `dktl init`.
 
-```bash
-mkdir my_project && cd my_project
-dktl dc up -d && dktl init
-dktl dkan:get <version_number>
-```
+    ├── dkan              # The upstream DKAN core codebase
+    ├── docroot           # Drupal core, and contrib modules not from DKAN
+    ├── src               # Site-specific configuration, code and files
```

Comment:
Let's also add the 'patches' directory here

Reply:
ok I'll add a mention of both _patches_ and _backup_